pkgs/applications/gis/qgis/3.0.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, lib, fetchurl, cmake, ninja, flex, bison, proj, geos, xlibsWrapper, sqlite, gsl
+, qwt, fcgi, python3Packages, libspatialindex, libspatialite, postgresql
+, qjson, txt2tags, openssl, libzip
+, qtbase, qtwebkit, qtsensors, qca-qt5, qtkeychain, qscintilla
+, withGrass ? true, grass
+}:
+with lib;
+let
+  pythonBuildInputs = with python3Packages;
+    [ python3Packages.qscintilla gdal jinja2 numpy psycopg2 pygments pyqt5 sip OWSLib six ];
```

Comment:
repeating `python3Packages`

Reply:
Not a repeat. The "with" does not cause the "qscintilla" to be shadowed. Without the extra python3Packages the "qscintilla" passed is not the one in python3Packages but the C++ library one.